src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -493,14 +490,14 @@ public void run() {
    private void abortActiveConnections(final ExecutorService assassinExecutor)
    {
       for (PoolEntry poolEntry : connectionBag.values(STATE_IN_USE)) {
+         Connection connection = poolEntry.close();
          try {
```

Comment:
...cancels leak task first. this pattern should be followed 'consistently'
proxyConnection.cancelLeakTask() looks like redundent / patchwork and can be removed with refactoring